texts/formatted_harry_potter.txt

```diff
@@ -0,0 +1,1679 @@
+"<Harry>"
+	"Harry" np ant m sg
+"<Potter>"
+;	"potter" n sg
+	"Potter" np cog sg
+"<is>"
+	"be" vbser pres p3 sg
+"<a>"
+	"a" det ind sg
+"<series>"
+	"series" n sg
+;	"series" n pl
+"<of>"
+	"of" pr
+"<fantasy>"
+	"fantasy" n sg
+"<novels>"
+	"novel" n pl
+"<written>"
+	"write" vblex pp
+;	"written" adj
+"<by>"
+;	"by" adv
+	"by" pr
+"<British>"
+	"British" adj
+;	"British" n sg
+;	"British" n pl
+"<author>"
+	"author" n sg
+;	"author" vblex inf
+;	"author" vblex pres
+;	"author" vblex imp
+"<J>"
+	"*J"
+"<.>"
+	"." sent
+"<K>"
+	"*K"
+"<.>"
+	"." sent
+"<Rowling>"
+	"Rowling" np cog sg
+"<.>"
+	"." sent
+"<The>"
+	"the" det def sp
+"<novels>"
+	"novel" n pl
+"<chronicle>"
+;	"chronicle" n sg
+;	"chronicle" vblex inf
+	"chronicle" vblex pres
+;	"chronicle" vblex imp
+"<the>"
+	"the" det def sp
+"<lives>"
+	"life" n pl
+;	"live" vblex pres p3 sg
+"<of>"
+	"of" pr
+"<a>"
+	"a" det ind sg
+"<young>"
+	"young" adj sint
+"<wizard>"
+	"wizard" n sg
+"<,>"
+	"," cm
+"<Harry>"
+	"Harry" np ant m sg
+"<Potter>"
+;	"potter" n sg
+	"Potter" np cog sg
+"<,>"
+	"," cm
+"<and>"
+	"and" cnjcoo
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<friends>"
+	"friend" n pl
+"<Hermione>"
+	"Hermione" np ant f sg
+"<Granger>"
+	"Granger" np cog sg
+"<and>"
+	"and" cnjcoo
+"<Ron>"
+	"Ron" np ant m sg
+"<Weasley>"
+	"Weasley" np cog sg
+"<,>"
+	"," cm
+"<all>"
+;	"all" adv
+;	"all" predet sp
+	"all" det qnt pl
+;	"all" prn qnt mf sp
```

Comment:
I think this should probably be pronoun (see tagging guidelines for "several" and "each").